chain/indexer.go

```diff
@@ -460,6 +469,7 @@ func (t *TipSetIndexer) stateChangedActors(ctx context.Context, old, new cid.Cid
 			return out, nil
 		}
 	}
+	log.Debugf("using slow state diff")
```

Comment:
should just be Debug, not sure if go-log will complain about the lack of print directives.